main.go

```diff
@@ -36,16 +36,14 @@ func main() {
 		Version = fmt.Sprintf("%s-%s", Version, VersionPostfix)
 	}
 
-	// NewVessel may take a payload for all suites to reference
-	pvtrVessel := pluginkit.NewEvaluationOrchestrator(PluginName, data.Loader, RequiredVars)
+	pvtrVessel := pluginkit.NewEvaluationOrchestrator(PluginName, nil, RequiredVars)
```

Comment:
We could also pass `data.Loader` here and pass `nil` below in `AddEvaluationSuite`, but this approach made more sense to me as the behavior of `NewEvaluationOrchestrator` with respect to its `loader` parameter is undocumented at present.

Reply:
@eddie-knight The choice I made here also notably removes the YAML output of the payload data. EvaluationOrchestrator.WriteResults was dumping the full payload along with the evaluation results previously.

I was not a fan of that behavior as it had the side effect of leaking the github credentials because the configuration was also dumped to YAML.

But, I'm calling out that side effect, as I guess its possible that it was viewed as desirable or directly depended upon by someone consuming that YAML output.